Modules/Loadable/Markups/MRML/CMakeLists.txt

```diff
@@ -26,3 +30,12 @@ SlicerMacroBuildModuleMRML(
   SRCS ${${KIT}_SRCS}
   TARGET_LIBRARIES ${${KIT}_TARGET_LIBRARIES}
   )
+
+#-----------------------------------------------------------------------------
+configure_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/../Resources/GenericStorage.markups.json
+  ${CMAKE_BINARY_DIR}/${Slicer_QTLOADABLEMODULES_SHARE_DIR}/${MODULE_NAME}/GenericStorage.markups.json
+  COPYONLY)
+install(
+  FILES ${CMAKE_BINARY_DIR}/${Slicer_QTLOADABLEMODULES_SHARE_DIR}/${MODULE_NAME}/GenericStorage.markups.json
```

Comment:
Could you confirm that we installing the file `GenericStorage.markups.json` only to be able to run the self test from within an installed slicer ?